Sources/Storage/Disk/DiskError.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  DiskError.swift
+//  StatsdClient
+//
+//  Created by Nghia Tran on 10/8/17.
+//  Copyright © 2017 StatsdClient. All rights reserved.
+//
+
+import Foundation
+
+enum DiskError: Error {
+
+    case fileNotFound
```

Comment:
`fileNotFound` & `invalidFile` are not used anywhere

Reply:
It tends for future.
So we shouldn't create it unless we use in a codebase, right?